### DISCUSSION TITLE
The solution to initiate Windows 10/11 Upgrade using Legacy Update

### DISCUSSION THREAD

**Discussion by sal-ammoniac** [LegacyUpdate/LegacyUpdate#272] [Ideas]
Directly downloading Windows 10/11 from LU will not initiate upgrade even after restart. Here is a solution.
1 Download Windows 10/11 Upgrade from LU webpage.
2 When it completed, do not restart. Find WindowsUpdateBox.exe in softwaredistribution folder and memrize the path
3 In cmd or anything else , run (path to WindowsUpdateBox)\WindowsUpdateBox /Update /Finalize /quiet
4 It will restart automatically and start upgrading.

(Tested on virtual machine and worked)

**Comment by kristibektashi** [comment]
Legacy Update should be changed to do this for Windows feature updates automatically when you select to restart for them
